providers/flagd/pom.xml

```diff
@@ -61,6 +61,12 @@
 			<version>6.0.53</version>
 			<scope>provided</scope>
 		</dependency>
+
+		<dependency>
+			<groupId>org.apache.commons</groupId>
+			<artifactId>commons-collections4</artifactId>
+			<version>4.4</version>
+		</dependency>
```

Comment:
`org.apache.commons` is a save group to include on the basis of it's popularity, glad you found an LRU impl there.